ruma-events/src/reaction.rs

```diff
@@ -27,11 +60,8 @@ impl ReactionEventContent {
     /// # Arguments
     ///
     /// * `event_id` - The id of the event we are reacting to.
-    /// * `key` - The key that indicates the reaction that is being applied,
-    /// usually an emoji.
-    pub fn new(event_id: EventId, key: String) -> Self {
-        ReactionEventContent {
-            relates_to: RelatesTo::Relation(Relation::Annotation(Annotation { event_id, key })),
-        }
+    /// * `emoji` - The emoji that indicates the reaction that is being applied.
```

Comment:
This is the actual emoji, right? Not a textual ID referring to one (like `entertained`). Not sure about wording but might be worth it to point this out.

Reply:
Yes, the actual emoji. Though nothing is stopping people from posting any string, not sure if we want to use emoji in the docs, add checks if the string is an emoji when deserializing or just continue using `key`.

Reply:
No, I don't think we should (try to) ensure it's just an emoji.